packages/ods/src/components/combobox/src/components/ods-combobox/ods-combobox.tsx

```diff
@@ -52,6 +52,7 @@ export class OdsCombobox implements OdsFormElement {
   @State() private hasNoResults: boolean = false;
   @State() private isInvalid: boolean | undefined;
   @State() private isOpen: boolean = false;
+  @State() private items: ResultElement[] = [];
```

Comment:
Looks like it is only meant to force a refresh of the element list when slot change.
Plus you just pass its ref to `this.resultElements` anyway.

If we do want to update the list without closing the list, we could just use the `forceUpdate` method to trigger a rerender.

I wonder even if it could be better to close and the reopen the dropdown when items change so that it isi visible by the user, otherwise the contents he is currently on, is suddenly modified (then the `isOpen` state will trigger the refresh)